### PR TITLE
Fix units

### PR DESCRIPTION
**Pull request by marcelofcastro**
Adding SI Units to some packages.

### CODE REVIEW

File: OpenIPSL/Electrical/Solar/KTH/Auxiliary/Capacitor.mo [hunk 1]
```diff
@@ -15,9 +15,9 @@ model Capacitor
         origin={-120.0,-0.0},
         extent={{-20.0,-20.0},{20.0,20.0}})));
   Real x(start=Udc0/(1000*UdcN));
```
Comment (dietmarw) [inline OpenIPSL/Electrical/Solar/KTH/Auxiliary/Capacitor.mo:17]: @marcelofcastro  Could it be that this factor 1000 also needs to get removed?

Comment (marcelofcastro) [reply OpenIPSL/Electrical/Solar/KTH/Auxiliary/Capacitor.mo:17]: good catch

**Review by dietmarw — APPROVED** [review]
I think this is good enough to merge.

**Comment by marcelofcastro** [comment]
Okay! Merging this PR.

**Pull request merged**